cypress/integration/ColorPicker.spec.ts

```diff
@@ -4,8 +4,27 @@ const getColorPicker = () => {
   return cy.get('[type="text"]');
 };
 
+const expandHex = (hex: string) => {
```

Comment:
not ideal, this is defined in `colorUtils.ts` but I didn't want to have to import from the main app/adjust the tsconfig